docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -957,6 +961,10 @@ include at build time.
 |Adds support for native memory tracking.
 |GraalVM for JDK 23 Mandrel 24.1
 
+|threaddump
+|Adds support for dumping thread stacks on SIGQUIT/SIGBREAK.
+|GraalVM for JDK 24 Mandrel 24.2
```

Comment:
Same here. We need to check support for this option. Especially when we add the option by default (which is a no for 23.1, so not sure how to proceed without accepting the fact that there won't be thread dumps in some version of quarkus that might still use 23.1).

I see that this has been considered for https://github.com/quarkusio/quarkus/pull/44000 as well, but since `nmt` isn't common it's a different can of worms.